src/lib/core/components/visualizations/implementations/ScatterplotVisualization.tsx

```diff
@@ -1227,13 +1262,34 @@ function ScatterplotWithControls({
     truncationConfigDependentAxisMax,
   } = useMemo(
     () =>
-      truncationConfig(defaultUIState, vizConfig, defaultDependentAxisRange),
+      truncationConfig(
+        {
+          independentAxisRange: xMinMaxDataRange,
+          dependentAxisRange: yMinMaxDataRange,
+        },
+        vizConfig,
+        {
+          // truncation overrides for the axis minima for log scale
+          // only pass values that you want to take effect!
```

Comment:
what do you mean by "take effect" here?

Reply:
I just mean that if you passed `{ truncationConfigIndependentAxisMin }` as the override object, then it would override if the value of `truncationConfigIndependentAxisMin` was true or false.  I've hopefully improved the comments.